package.json

```diff
@@ -57,11 +57,11 @@
     "@babel/preset-env": "7.14.9",
     "@justeat/browserslist-config-fozzie": "1.2.0",
     "@justeat/eslint-config-fozzie": "5.1.0",
-    "@justeat/fozzie": "8.0.0",
     "@justeat/f-wdio-utils": "0.8.1",
+    "@justeat/fozzie": "8.0.0",
```

Comment:
Not sure why this got moved around in the file... It won't impact anything though as the version is still the same.